src/Value.js

```diff
@@ -17,7 +17,7 @@ var Option = React.createClass({
 	render: function() {
 		return (
 			<div className="Select-item">
-				<span className="Select-item-icon" onMouseDown={this.blockEvent} onClick={this.props.onRemove}>&times;</span>
+				<span className="Select-item-icon" onMouseDown={this.blockEvent} onClick={this.props.onRemove} onTouchEnd={this.props.onRemove}>&times;</span>
```

Comment:
Shouldn't the `onClick` handler still fire here on a touch device? or is it blocked by the `onMouseDown` handler?